build.gradle

```diff
@@ -1,3 +1,8 @@
+plugins {
+  id "org.owasp.dependencycheck" version "1.4.5"
+  id "com.github.johnrengelman.shadow" version "1.2.4"
```

Comment:
Specifying it using the plugins block makes it download the dependencies from plugins.gradle.org... Specifically here: https://plugins.gradle.org/plugin/org.owasp.dependencycheck

There is also an option of the longer buildscript{} if you wanted to get dependencies the old way..